### PR TITLE
Deprecates stationstation.yml

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Goodbye old friend.

### CODE REVIEW

**Review by metalgearsloth — APPROVED** [review]
image

**Review by DrSmugleaf — APPROVED** [review]
2f0

**Comment by DrSmugleaf** [comment]
Tests that use stationstation.yml need to be updated for this PR.

**Comment by SweptWasTaken** [comment]
> 
> 
> Tests that use stationstation.yml need to be updated for this PR.

I don't have access so I cannot do that.

**Comment by metalgearsloth** [comment]
I tried removing it from tests but some weird shit was going on with power and save/load test.

**Comment by Erisfiregamer1** [comment]
Try, um, updating the PR?

**Comment by SweptWasTaken** [comment]
Merged with master. It's not failing the tests anymore but that's because the tests are broken globally on the repository.

**Comment by metalgearsloth** [comment]
I did some digging:

There's a test that specifically loads stationstation to make sure it hasn't changed. I changed it to saltern but it does in fact change as the map isn't set as paused because power is still running during the test.

tl;dr is that it needs https://github.com/space-wizards/RobustToolbox/pull/1531
